### ISSUE TITLE
ETC remote config

### ISSUE THREAD

**Issue by hey000hey** [UnamSanctam/UnamWebPanel#205]
Is this the correct format? I copied-pastd it from the builder, but it's getting 0 h/s when I use the config remotely (changing it to default ethminer works). The wallet/pool is mixed into one option, so I'm a bit confused.

https://rentry.co/2rfvn

**Comment by UnamSanctam** [comment]
Yes that's the correct format for ETC, it should work fine. What exactly is it saying in in the web panel?

**Comment by hey000hey** [comment]
Well, it doesn't show any errors, but I just get 0H/s, while setting it to default ethminer works.

**Comment by hey000hey** [comment]
![image](https://user-images.githubusercontent.com/83317718/200210998-c734a94b-2f12-419a-9026-0ddf39614870.png)
everything else works except the one I set to the config

**Comment by UnamSanctam** [comment]
And its status is Active? How long do you wait after switching the configuration? Because I tried the configuration you linked inside my web panel and it works fine there (miner version 3.1.0).
